src/main/java/presenter/LoginResponsePresenter.java

```diff
@@ -1,4 +1,4 @@
-package Presenter;
+package presenter;
 
 import Entity.User;
```

Comment:
entity

Reply:
Oh this is actually Josh's part, I accidentally changed it, oops!